src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorCreateTest.java

```diff
@@ -155,9 +151,9 @@ void shouldValidateApplicant1SolicitorOrgAndReturnNoErrorsWhenSolicitorBelongsTo
     public void shouldValidateApplicant1SolicitorOrgAndReturnErrorWhenSolicitorDoesNotBelongsToSelectedOrg() throws Exception {
         when(serviceTokenGenerator.generate()).thenReturn(TEST_SERVICE_AUTH_TOKEN);
 
-        stubGetOrganisationEndpoint("TESTORG123", objectMapper);
+        prdOrganisationUtil.stubGetOrganisationEndpoint("TESTORG123");
```

Comment:
`documentAssemblyUtil`  `prdOrganisationUtil` can have  `stubForDocAssembly` methods static so you won't need to auto wire them ?

Reply:
Reverted to static and removed the need for the ObjectMapper when stubbing